packages/docs/.storybook/preview.ts

```diff
@@ -70,7 +72,102 @@ const preview: Preview = {
       toc: {
         headingSelector: 'h2, h3',
       },
-      source: { format: 'html' }
+      source: {
+        format: 'html', transform: (code: string, storyContext: StoryContext) => {
```

Comment:
I think it would be nicer / a little bit more clearer, if we would outsource this functionality into its own file, so the `preview.ts` will not become too overloaded